src/webgpu/shader/validation/wgsl/runtime-array-is-store-type.fail.wgsl

```diff
@@ -0,0 +1,12 @@
+# v-0031 - This fails because the runtime array must not be the store type of variable 'buf'.
```

Comment:
This sentence doesn't seem quite right, the store type of `buf` is `Foo` not the runtime array.